docs/example-scenario/infrastructure/regulated-multitier-app.md

```diff
@@ -80,7 +80,7 @@ For general guidance on designing resilient scenarios, see [Designing resilient
 **Prerequisites.**
 
 * You must have an existing Azure account. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/?WT.mc_id=A261C142F) before you begin.
-* To deploy a SQL Server cluster into the backend scale set, you would need an Active Directory Directory Services domain.
+* To deploy a SQL Server cluster into the backend scale set, you would need an Active Directory Services domain.
```

Comment:
This should be -> "a domain in Azure Active Directory (AD) Domain Services."

Reply:
changed